src/ship.class.ts

```diff
@@ -47,6 +48,53 @@ export class Ship {
             .easing(TWEEN.Easing.Linear.None);
     }
 
+    /**
+     * Loads (fill) the ship.
+     */
+    load(): void {
+        this._empty = false;
+        this.drawFullShip();
+    }
+
+    /**
+     * Unloads (empty) the ship.
+     */
+    unload(): void {
+        this._empty = true;
+        this.drawEmptyShip();
+    }
+
+    /**
+     * Returns flag whether the ship is empty.
+     */
+    get empty(): boolean {
```

Comment:
implement the class with interface `Container` to describe loading functionality; this should be used to `Ship` and `Dock` classes

Reply:
Resolved with commits:
* 67e06ce3230572c5eb6b474dc22d5fac1b0ee787
* 6995de23c6f6fdf33e59df8e0a56112af9c3cb82
* 0c0881a288d4ea1f0760b3757c5254c4c3f843c7